api/src/routes/challenge.ts

```diff
@@ -403,26 +402,35 @@ export const challengeRoutes: FastifyPluginCallbackTypebox = (
         const user = await fastify.prisma.user.findUniqueOrThrow({
           where: { id: req.session.user.id }
         });
+        const savableChallenges = user.savedChallenges;
```

Comment:
This is the currently saved challenges, not the challenges that can be saved. Those are challenges with type `multifileCertProject`

Reply:
yeab https://github.com/Sboonny/freeCodeCamp/blob/a19535171796fa8103ceb640fe86d3321419c544/api-server/src/server/boot/challenge.js#L127-L129, I am trying to figure out why the body isn't updated after the endpoint has run.